tools/c7n_azure/tests/azure_common.py

```diff
@@ -190,6 +191,7 @@ def _response_callback(self, response):
         body = AzureVCRBaseTest._replace_tenant_id(body)
         body = AzureVCRBaseTest._replace_subscription_id(body)
         body = AzureVCRBaseTest._replace_storage_keys(body)
+        body = AzureVCRBaseTest._replace_instrumentation_key(body)
```

Comment:
At some point I guess we should just lift the whole azure common test bits - they do a more robust plugin model here to register cassette modifications like this.